cli/src/cli_util.rs

```diff
@@ -1591,7 +1593,9 @@ See https://martinvonz.github.io/jj/latest/working-copy/#stale-working-copy \
             .map(|commit_id| tx.repo().store().get_commit(commit_id))
             .transpose()?;
 
-        if self.working_copy_shared_with_git {
+        if self.working_copy_shared_with_git() {
```

Comment:
Nit: you already have `working_copy_shared_with_git` initialized with the value above, so reuse it here.